charts/ccsm-helm/README.md

```diff
@@ -238,9 +238,47 @@ Information about Tasklist you can find [here](https://docs.camunda.io/docs/comp
 | | `ingress.enabled` | If true, an ingress resource is deployed with the Tasklist deployment. Only useful if an ingress controller is available, like nginx. | `false` |
 | | `ingress.className` | Defines the class or configuration of ingress which should be used by the controller | `nginx` |
 | | `ingress.annotations` | Defines the ingress related annotations, consumed mostly by the ingress controller | `ingress.kubernetes.io/rewrite-target: "/"` <br/> `nginx.ingress.kubernetes.io/ssl-redirect: "false"` |
-| | `ingress.path` | Defines the path which is associated with the Tasklist service and port https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules | `/` |
+| | `ingress.path` | Defines the path which is associated with the Tasklist [service and port](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules) | `/` |
 | | `ingress.host` | Can be used to define the [host of the ingress rule.](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-rules) If not specified the rules applies to all inbound HTTP traffic, if specified the rule applies to that host. | `""` |
 
+### Identity
+
+Information about Identity you can find [here](https://docs.camunda.io/docs/self-managed/identity/what-is-identity/).
```

Comment:
:question: Depends on https://github.com/camunda-cloud/camunda-cloud-documentation/pull/689, do we want to add the link already?

Reply:
Yes see https://camunda.slack.com/archives/CRQAL5A1M/p1648559459745149?thread_ts=1648559134.417009&cid=CRQAL5A1M